content/docs/user-guide/managing-external-data.md

```diff
@@ -66,21 +66,23 @@ $ dvc run -d data.txt \
 
 ```dvc
 # Add SSH remote to be used as cache location for SSH files
-$ dvc remote add sshcache ssh://user@example.com:/cache
+$ dvc remote add sshcache ssh://user@example.com/home/.../cache
 
 # Tell DVC to use the 'sshcache' remote as SSH cache location
 $ dvc config cache.ssh sshcache
 
 # Add data on SSH directly
-$ dvc add --external ssh://user@example.com:/mydata
+$ dvc add --external ssh://user@example.com/home/.../mydata
 
 # Create the stage with an external SSH output
 $ dvc run -d data.txt \
           --external \
-          -o ssh://user@example.com:/home/shared/data.txt \
-          scp data.txt user@example.com:/home/shared/data.txt
+          -o ssh://user@example.com/home/shared/data.txt \
+          scp data.txt user@example.com/home/shared/data.txt
```

Comment:
I'm not sure that this is a valid scp url, I would rather not touch it.

Reply:
Good catch, thanks. But we do want to remove `:` from the SSH URL above, right?

Reply:
alright, I assume those `:` are OK to have been removed 🙂